catalog/spatial/wfs/2.0.0/spatial-wfs-v2_0_0-common/src/test/java/org/codice/ddf/spatial/ogc/wfs/v2_0_0/catalog/common/TestFeatureMetacardType.java

```diff
@@ -463,7 +473,9 @@ public void testFeatureMetacardTypeFindBasicMetacardAttributes() {
                 NON_QUERYABLE_PROPS,
                 Wfs20Constants.GML_3_2_NAMESPACE);
 
-        for (AttributeDescriptor ad : BasicTypes.BASIC_METACARD.getAttributeDescriptors()) {
+        Set<AttributeDescriptor> descriptors = initDescriptors();
+
+        for (AttributeDescriptor ad : descriptors) {
```

Comment:
It will be the descriptors size + 1 (the `ELEMENT_NAME` added before). I will add an assertion for this.